README.md

```diff
@@ -1,282 +1,37 @@
-
-```yaml
-unattended_systemd_timer_override: true
-# apt-daily timer
-unattended_apt_daily_oncalendar: "*-*-* 00:30"
-unattended_apt_daily_randomizeddelaysec: "60m"
-
-# apt-daily-upgrade timer
-unattended_apt_daily_upgrade_oncalendar: "*-*-* 4:00"
-unattended_apt_daily_upgrade_randomizeddelaysec: "90m"
-```
-
-## Role Usage Examples
-
-Example for Ubuntu, with custom [origins patterns](#patterns-examples), blacklisted packages and e-mail notification:
-
-```yaml
-- hosts: all
-  roles:
-  - role: hifis.unattended_upgrades
-    unattended_origins_patterns:
-    - 'origin=Ubuntu,archive=${distro_codename}-security'
-    - 'o=Ubuntu,a=${distro_codename}-updates'
-    unattended_package_blacklist: [cowsay, vim]
-    unattended_mail: 'root@example.com'
-```
-
-_Note:_ You don't need to specify `unattended_origins_patterns`, the role will use distribution's default if the variable is not set.
-
-### Running Only on Debian-based Systems
-
-If you manage multiple distribution with the same playbook, you may want to skip running this role on non-Debian systems. You can [use `when` conditional with role](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#conditionals-with-roles) to limit the role to particular systems:
-
-```yaml
-- hosts: all
-  roles:
-     - role: hifis.unattended_upgrades
-       when: ansible_facts['os_family'] == 'Debian'
-```
-
-See [#38](https://github.com/jnv/ansible-role-unattended-upgrades/pull/38) for discussion.
-
-### Patterns Examples
-
-By default, only security updates are allowed for both Ubuntu and Debian. You can add more patterns to allow unattended-updates install more packages automatically, however be aware that automated major updates may potentially break your system.
-
-#### For Debian
-
-```yaml
-unattended_origins_patterns:
-  - 'origin=Debian,codename=${distro_codename},label=Debian-Security' # security updates
-  - 'o=Debian,codename=${distro_codename},label=Debian' # updates including non-security updates
-  - 'o=Debian,codename=${distro_codename},a=proposed-updates'
-```
-
-#### For Ubuntu
-
-In Ubuntu, archive always contains the distribution codename
-
-```yaml
-unattended_origins_patterns:
-  - 'origin=Ubuntu,archive=${distro_codename}-security'
-  - 'o=Ubuntu,a=${distro_codename}'
-  - 'o=Ubuntu,a=${distro_codename}-updates'
-  - 'o=Ubuntu,a=${distro_codename}-proposed-updates'
-```
-
-#### For Raspbian
-
-In Raspbian, it is only possible to update all packages from the default repository, including non-security updates, or updating none.
-
-Updating all, including non-security:
-
-```yaml
-unattended_origins_patterns:
-  - 'origin=Raspbian,codename=${distro_codename},label=Raspbian'
-```
-
-To not install any updates on a raspbian host, just set `unattended_origins_patterns` to an empty list:
-
-```yaml
-unattended_origins_patterns: []
+```shell
+ansible-galaxy collection install hifis.software_services
 ```
 
 ## License
 
 GPL-2.0-or-later
-
-## Author
```

Comment:
Shall we keep the author section and state that "This Ansible collection is maintained by [HIFIS Software Services](https://www.hifis.net/)."?

Reply:
:+1: